package.json

```diff
@@ -626,6 +627,60 @@
                                         ]
                                     }
                                 }
+                            },
+                            "python": {
```

Comment:
We need to schedule work to update the documentation. We probably need a Python "tutorial" topic plus update the documentation for build & debug tasks.

Reply:
Thanks Karol. Absolutely, I'll work with Uche to get that work tracked.